cmd/commands/cli/command_cli.go

```diff
@@ -53,6 +53,8 @@ const statusConnected = "Connected"
 
 // Run runs CLI interface synchronously, in the same thread while blocking it
 func (c *Command) Run() (err error) {
+	license := cmd.GetStartupLicense("type `show w'", "type `show c'")
```

Comment:
Looks like `show w` is wrong recomendation

Reply:
Fixed